engines/wintermute/detection_tables.h

```diff
@@ -68,8 +68,8 @@ static const PlainGameDescriptor wintermuteGames[] = {
 	{"pigeons",         "Pigeons in the Park"},
 	{"projectdoom",     "Project: Doom"},
 	{"projectjoe",      "Project Joe"},
-	{"reversion1",      "Reversion: The Escape"},
-	{"reversion2",      "Reversion: The Meeting"},
+	{"reversion1",      "Reversion: Chapter 1 - The Escape"},
```

Comment:
I based this on the game titles how they appear in the Steam store.

See http://store.steampowered.com/app/270570/ and http://store.steampowered.com/app/281060/ for reference.